composer/trainer/trainer.py

```diff
@@ -839,12 +839,22 @@ def __init__(
         # Console Logging
         loggers = list(ensure_tuple(loggers))
         if any(isinstance(x, ProgressBarLogger) for x in loggers):
-            warnings.warn(
-                DeprecationWarning(
-                    (f'Specifying the {ProgressBarLogger.__name__} via `loggers` is deprecated. Instead, '
-                     'please specify `progress_bar`, `log_to_console`, `log_level`, and `stream` arguments when '
-                     'constructing the trainer. If specified, these arguments will be ignored, as the '
-                     f'{ProgressBarLogger.__name__} was already created.')))
+            if progress_bar != True:
```

Comment:
I think we want to preserve the deprecation warning, or -- since we are moving to a major version release, we can just throw an error here if `ProgressBarLogger` is provided?

Reply:
Going based off [this](https://www.notion.so/Customizable-Progress-Bar-ee795681d6624aa2b07b6ccff5cae8cf#074f0b8b492a4d5c8a1b21996ad0580f). Previously, this option was deprecated, but we're now un-deprecating it and supporting creating ProgressBarLogger in `loggers`. See example YAMLs at top of PR.